### ISSUE TITLE
E key is not moving to the next issue

### ISSUE THREAD

**Issue by damithc** [HubTurbo/HubTurbo#1269]
`E` is used to mark the issue as read and move to the next issue. 
It used to work like that but it stopped working a few minutes ago. It still marks the issue as read, but doesn't go to the next issue automatically.
Log attached.
[hubturbo-log - Copy.zip](https://github.com/HubTurbo/HubTurbo/files/132523/hubturbo-log.-.Copy.zip)

**Comment by damithc** [comment]
Started working again after a restart.

**Comment by damithc** [comment]
It seems the functionality is working but sometimes it takes a very long time. e.g. 20 seconds. This may be due to an ongoing update in the background.

**Comment by damithc** [comment]
Nope, sometimes it doesn't work even after waiting for a long time. Possibly due to a mess up in the card indexes?

**Comment by acruis** [comment]
Do other key shortcuts work when E fails?

**Comment by damithc** [comment]
Yes. Even `E` works partially. It's the 'move to next card automatically` part that is not working. The jumping is probably based on an internal index and may be there is an 'off by one' error causing HT to jump to the same card? Just guessing.

**Comment by acruis** [comment]
Yes, most likely a problem with `NavigableListView.selectNextItem()`
